src/adminBase.js

```diff
@@ -1,43 +1,49 @@
 /* 
 * @Author: Mike Reich
 * @Date:   2016-02-05 15:38:26
-* @Last Modified 2016-02-12
+* @Last Modified 2016-02-15
 */
 
 'use strict';
 
 import {HasModels} from '@nxus/storage'
 import pluralize from 'pluralize'
-
+import capitalize from 'capitalize'
+import _ from 'underscore'
+import morph from 'morph'
 
 /**
  * The AdminBase class provides a set of helper CRUD classes for defining Admin-UI based admin pages.
  */
 export default class AdminBase extends HasModels {
 
-  constructor(app) {
+  constructor(app, opts) {
```

Comment:
So this is a pretty big change, is it your intention that folks would not use AdminBase directly anymore (as a superclass) but only via `admin.adminModel`? We've got several cases in MHL where this is a base class because the subclasses need to override e.g. the save implementation.